docs/start/install.rst

```diff
@@ -1,129 +1,279 @@
 
+    $ pip install pip --upgrade
+    Collecting pip
+      Using cached https://files.pythonhosted.org/.../pip-20.1.1-py2.py3-none-any.whl
+    Installing collected packages: pip
+      Found existing installation: pip 19.3.1
+        Uninstalling pip-19.3.1:
+          Successfully uninstalled pip-19.3.1
+    Successfully installed pip-20.1.1
 
+There are no other prerequisites. Datatable does not depend on any other python
+module [#v11]_, nor on any non-standard system library.
 
-Install on macOS and Linux
------------------------------
 
-Run the following command to install ``datatable`` on macOS and Linux:
+
+Basic installation
+------------------
+
+On most platforms ``datatable`` can be installed directly from `PyPI`_ using
+``pip``:
 
 .. xcode:: shell
 
-  $ pip install datatable
+    $ pip install datatable
 
+The following platforms are supported:
 
+- **macOS**
 
-Install on Windows
-------------------
+  Datatable has been tested to work on macOS 10.12.5 (Sierra), macoS 10.13.6
+  (High Sierra), and macOS 10.15.2 (Catalina).
+
+- **Linux x86_64 / ppc64le**
+
+  We produce binary wheels that are tagged as ``manylinux2010`` (for x86_64
+  architecture) and ``manylinux2014`` (for ppc64le). Consequently, they will
+  work with your Linux distribution if it is compatible with one of these tags.
+  Please refer to `PEP-513`_ and `PEP-599`_ for details.
+
+- **Windows (N/A)**
+
+  Windows wheels cannot be installed from PyPI just yet, but they will be
+  available after the next release. In the meanwhile, datatable development
+  wheels can be installed (see section below).
+
+
+
+Install latest dev version
+--------------------------
 
-As of version 0.11.0, ``datatable`` will also be available on Windows,
-and the installation process will be identical to the process
-for macOS and Linux.
+If you wish to test the latest version of ``datatable`` before it has been
+officially released, then you can use one of the binary wheels that we build
+as part of our Continuous Integration process.
 
-Meanwhile, you can install the dev version of ``datatable`` on Windows
-from the pre-built wheels that are available on
-`AppVeyor <https://ci.appveyor.com/project/h2oops/datatable/history>`__.
-To do so, simply click on the master build of your choice and
-then navigate to ``Artifacts``. Copy the wheel URL that corresponds
-to your version of Python and then install it as:
+If you are on Windows, then pre-built wheels are available on `AppVeyor`_.
+Click on a green master build of your choice, then navigate to the "Artifacts"
+tab, copy the wheel URL that corresponds to your Python version, and finally
+install it as:
 
 .. xcode:: winshell
 
-   C:\> pip install YOUR_WHEEL_URL
+    C:\> pip install YOUR_WHEEL_URL
 
+For macOS and Linux, development wheels can be found at our `S3 repository`_.
+Scroll to the bottom of the page to find the latest links, and then download
```

Comment:
Have you been thinking about adding builds to the `index.html` file in a reverse order? Then one won't need to scroll all the time.

Reply:
It's much easier (and more reliable) to append text at the bottom of the page, than insert somewhere in the middle. Plus, it's not like this is such a common way of installation anyways...